client/network/src/service.rs

```diff
@@ -733,25 +733,108 @@ pub struct NetworkWorker<B: BlockT + 'static, H: ExHashT> {
 	/// Senders for events that happen on the network.
 	event_streams: Vec<mpsc::UnboundedSender<Event>>,
 	/// Prometheus network metrics.
-	metrics: Option<Metrics>
+	metrics: Option<Metrics>,
 }
 
 struct Metrics {
+	// This list is ordered alphabetically
+	connections: Gauge<U64>,
+	import_queue_blocks_submitted: Counter<U64>,
+	import_queue_finality_proofs_submitted: Counter<U64>,
+	import_queue_justifications_submitted: Counter<U64>,
```

Comment:
I think it might be better to put these all into one `CounterVec<U64>`.

Reply:
I'm not sure about this one, since we're measuring different things. It for example doesn't make sense to make the sum or the average of these three values.

Reply:
Hmm, yeah not sure. @mxinden?

Reply:
Using a `CounterVec` here would ease displaying both finality proofs as well as justifications in the same graph within the Prometheus UI. Once one goes through Grafana it doesn't matter as Grafana can display arbitrary time series within the same graph.